manifests/init.pp

```diff
@@ -203,6 +203,8 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $yum_max_speed::              Maximum bandwidth used per download in bytes/sec
```

Comment:
I would add `type:integer`, and include @ekohl's suggestion validation, but otherwise this PR looks good to me